README.md

```diff
@@ -2,16 +2,19 @@
 
 # Project Name
 
-> Description the project.
+> One paragraph statement about the project.
 
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
 
 ## Built With
 
 - Major languages
 - Frameworks
 - Technologies used
```

Comment:
Rember to add the languages, frameworks, or technologies used to build your project, like `HTML`, `CSS`,...